frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -1007,6 +1008,15 @@ function BookInfo:expandString(str, file, timestamp)
                         patterns["%C"] = title
                     end
                 end
+                if patterns["%l"] then
+                    local pages_left_in_chapter = self.ui.toc:getChapterPagesLeft(pageno) or pages_left_in_chapter
+                    if pages_left_in_chapter and pages_left_in_chapter ~= "" then
```

Comment:
This one's always a number or nil.
```suggestion
                    local pages_left_in_chapter = self.ui.toc:getChapterPagesLeft(pageno) or pages_left_in_chapter
                    if pages_left_in_chapter then
```